tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -123,13 +123,6 @@ public class TestProgram : IDisposable
                     .AddDatabase(mongoDbName);
                 IntegrationServiceABuilder = IntegrationServiceABuilder.WithReference(mongodb);
             }
-            if (!resourcesToSkip.HasFlag(TestResourceNames.oracledatabase))
```

Comment:
Can this line be removed now?

https://github.com/dotnet/aspire/blob/21ca4899c4f28926856783a465c02f4bdf606e11/tests/testproject/TestProject.AppHost/TestProject.AppHost.csproj#L21